source/includes/_wallet_accounts.md

```diff
@@ -0,0 +1,220 @@
+# Wallet Accounts
+## Account Object
+> An account object looks like this:
+
+```json
+  {
+    "wid": 1,
+    "id": "test",
+    "name": "default",
+    "initialized": true,
+    "witness": false,
+    "watchOnly": false,
+    "type": "pubkeyhash",
+    "m": 1,
+    "n": 1,
+    "accountIndex": 0,
+    "receiveDepth": 8,
+    "changeDepth": 1,
+    "nestedDepth": 0,
+    "lookahead": 10,
+    "receiveAddress": "mu5Puppq4Es3mibRskMwoGjoZujHCFRwGS",
+    "nestedAddress": null,
+    "changeAddress": "n3nFYgQR2mrLwC3X66xHNsx4UqhS3rkSnY",
+    "accountKey": "tpubDC5u44zLNUVo2gPVdqCbtX644PKccH5VZB3nqUgeCiwKoi6BQZGtr5d6hhougcD6PqjszsbR3xHrQ5k8yTbUt64aSthWuNdGi7zSwfGVuxc",
+    "keys": []
+  } 
+
+```
+Represents a BIP44 Account belonging to a Wallet.
+Note that this object does not enforce locks. Any method that does a write is internal API only and will lead to race conditions if used elsewhere.
+
+From the [BIP44 Specification](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki):
+
+<p>
+    <code>
+        This level splits the key space into independent user identities, so the wallet never mixes the coins across different accounts.
+    </code>
+</p>
+<p>
+    <code>
+        Users can use these accounts to organize the funds in the same fashion as bank accounts; for donation purposes (where all addresses are considered public), for saving purposes, for common expenses etc.
+    </code>
+ </p>
+ <p>
+    <code>
+        Accounts are numbered from index 0 in sequentially increasing manner. This number is used as child index in BIP32 derivation.
+    </code>
+</p>
+<p>
+    <code>
+        Hardened derivation is used at this level.
+    </code>
+</p>
+
+## Get Wallet Account List
+
+```shell--vars
+id='test' 
+```
+
+```shell--cli
+bcoin cli wallet account list --id=$id
+```
+
+```javascript
+const client = new bcoin.http.Client();
+(async () => {
+  const accountInfo = await client.getAccounts(id);
+  console.log(accountInfo);
+})();
```

Comment:
This would technically be true for all of these though right? Node and I were originally thinking about how to cut down on just having a ton of repeated code, but this might be somewhere where it makes sense to avoid a lot of unnecessary support requests. Maybe we'll do this in a separate PR to add back to all the functions? Could probably be done with a script actually...

Reply:
Correct, I just thought I'd mention it there.